debian/changelog

```diff
@@ -0,0 +1,5 @@
+stocks (1.0-1) stable; urgency=medium
```

Comment:
No, this is not the case. The stocks _source_ package contains both the server and client code, and since these versions diverge I chose to use 1.0-1 here and will increment this version whenever we are cutting a Debian release.